admin_guide/managing_networking.adoc

```diff
@@ -420,50 +421,82 @@ $ oc create -f <replication_controller>.json
 oc describe rc <replication_controller>
 ----
 
-[[multicast]]
+[[admin-guide-networking-multicast]]
 == Enabling Multicast
 
-Normally, multicast traffic between pods is disabled in {product-title}, but you
-can enable it on a per-project basis by setting an annotation on the project's
-corresponding NetNamespace object:
-
+[IMPORTANT]
+====
+Enabling multicast networking is a Technology Preview only. Technology
+Preview features are not supported with Red Hat production service level
+agreements (SLAs), might not be functionally complete, and Red Hat does not
+recommend to use them for production. These features provide early access to
```

Comment:
this phrasing seems odd to me "recommend to use them for production."

I like the wording in the official statement "are not intended for production use"